packages/react-native-bridge/ios/RNReactNativeGutenbergBridge.swift

```diff
@@ -421,11 +421,23 @@ public class RNReactNativeGutenbergBridge: RCTEventEmitter {
     func toggleRedoButton(_ isDisabled: Bool) {
         self.delegate?.gutenbergDidRequestToggleRedoButton(isDisabled)
     }
+    
+    @objc
+    func requestConnectionStatus(_ callback: @escaping RCTResponseSenderBlock) {
+        callback([self.delegate?.gutenbergDidRequestConnectionStatus() ?? true])
+    }
 
-	@objc
-	func requestConnectionStatus(_ callback: @escaping RCTResponseSenderBlock) {
-		callback([self.delegate?.gutenbergDidRequestConnectionStatus() ?? true])
-	}
+    @objc
+    func logException(_ rawException: [AnyHashable: Any], callback: @escaping RCTResponseSenderBlock) {
+        guard let exception = GutenbergJSException(from: rawException) else {
+            callback([false])
+            return
+        }
+    
+        self.delegate?.gutenbergDidRequestLogException(exception) {
```

Comment:
Follow up of [this comment](https://github.com/WordPress/gutenberg/pull/59221/files/c017e122aa58dbfcc5e33fad72a1b9207ac77880#r1507239536), what do you think removing the callback argument in the delegate function (and the one in Tracks)? This line would become:

```swift
self.delegate?.gutenbergDidRequestLogException(exception)
callback([true])
```

IMO, we don't need to put this callback into Tracks, and calling `callback` synchronously doesn't appear to have any difference compared to calling it asynchronously.